openapi/v2/device-sdk.yaml

```diff
@@ -173,48 +145,43 @@ components:
       description: "A profile defining a class of device to be onboarded, including its capabilities and data format."
       type: object
       properties:
-        created:
-          type: integer
-          description: "A Unix timestamp indicating when the object was initially persisted to a database"
-        modified:
-          type: integer
-          description: "A Unix timestamp indicating when the object was last modified"
-        description:
+        apiVersion:
```

Comment:
apiVersion will only be in request, response, and event

Reply:
Removed the `apiVersion ` from the device profile schema.